src/test/java/org/opensearch/security/dlic/rest/api/AuditApiActionValidationTest.java

```diff
@@ -51,17 +50,6 @@ public void enabledAuditApi() {
         }
     }
 
-    @Test
-    public void configEntityNameOnly() {
```

Comment:
should we add a test here that tests for route not found?

Reply:
We have tests for such behave. In `AuditApiActionTest` and `SecurityConfigApiTest`